### ISSUE TITLE
\CrowdinApiClient\Api\TranslationApi::uploadTranslations missing content-type header

### ISSUE THREAD

**Issue by seggewiss** [crowdin/crowdin-api-client-php#66]
The \CrowdinApiClient\Api\TranslationApi::uploadTranslations function is missing the content-type "application/json" header.
This results in a 415 unsupported media type response.
Steps to reproduce:

1. Create a storage e.g:
    1.1 `$storage = $this->crowdin->storage->create(new \SplFileObject($file));` 
2. Try to upload the storage as a translation file e.g:
```
$this->crowdin->translation->uploadTranslations($crowdinProjectId, $languageId, [
    'storageId' => $storage->getId(),
    'fileId' => $fileId,
]); 
```

**Comment by andrii-bodnar** [comment]
Hi @seggewiss!

We have just released a new version [1.4.2](https://github.com/crowdin/crowdin-api-client-php/releases/tag/1.4.2) with the fix. Could you please check if everything is ok now?

**Comment by seggewiss** [comment]
Hi @andrii-bodnar 
Works like a charm. Thanks for your quick fix :smiley:
